prism/src/main/java/org/apache/falcon/util/SecureEmbeddedServer.java

```diff
@@ -48,6 +48,13 @@ protected Connector getConnector(int port) {
         connector.setPassword(properties.getProperty("password",
                 System.getProperty("password", "falcon-prism-passwd")));
         connector.setWantClientAuth(true);
+
+        // this is to enable large header sizes when Kerberos is enabled with AD
+        final Integer bufferSize = Integer.valueOf(StartupProperties.get().getProperty(
+                "falcon.jetty.request.buffer.size", "16192"));
```

Comment:
We should add this property to startup.properties as well with default value and a comment to describe the property. This will enable users to know that to tweak this behavior they can change value of this property.